test/typescript-dts/grid/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+    "compilerOptions": {
+        "esModuleInterop": false,
+        "noEmit": true,
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictFunctionTypes": true,
+        "strictNullChecks": true,
+        "baseUrl": "../../../",
+        "module": "umd",
+        "moduleResolution": "node",
+        "rootDir": "../../../",
+        "lib": ["es5", "dom"],
+        "paths": {
+            "@highcharts/grid": ["code/grid"],
+            "@highcharts/grid/*": ["code/grid/*"]
+        },
+        "types": []
+    },
+    "exclude": [
+        "../../../node_modules"
+    ]
+}
```

Comment:
Make use of extends and move the `paths` setting to the upper tsconfig. Why is `exclude` necessary? This is a default in the TypeScript compiler.